common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala

```diff
@@ -40,6 +37,10 @@ import java.time.Instant
 import scala.collection.immutable.SortedMap
 
 object ObsQueries {
+  implicit class ListOps[A](val list: List[A]) {
+    def toSortedMap[K: Ordering, V](getKey: A => K, getValue: A => V = identity[A](_)) =
```

Comment:
Why can't we use the one here https://github.com/gemini-hlsw/explore/blob/master/common/src/main/scala/explore/implicits.scala#L64 ?

Reply:
I think right now the `common` module depends on this new module.

Reply:
I moved it now to `model`